src/main/java/net/coco/pieces/PawnEnum.java

```diff
@@ -0,0 +1,15 @@
+package net.coco.pieces;
+
+public enum PawnEnum {
+    WHITE('p'), BLACK('P');
```

Comment:
```suggestion
public enum PawnEnum {
    WHITE('p'), 
    BLACK('P');
```
Enum의 필드는 한 줄에 하나씩 입력해주세요~

Reply:
수정 완료했습니다!